### ISSUE TITLE
feat: Scheduler

### ISSUE THREAD

**Issue by ghost** [KumarAbhirup/bulk-mail-cli#28]
I'd like my mails to only be sent at certain hours for example: between 9 AM and 5 PM.

**Comment by kumarabhirup** [comment]
The bulk-mail-cli configuration has an `interval` config that can take a cron setting.

So what you want is technically achievable in bulk-mail-cli v2.

```json
{
...
"configuration": {
    "mailInterval": "*/10 * * * * *", // Cron Expression: To mail every 10 seconds [ >= 10 seconds recommended ]
  }
...
}
```

Reference Link: [https://stackoverflow.com/questions/41743720/crontab-run-every-15-minutes-between-certain-hours](https://stackoverflow.com/questions/41743720/crontab-run-every-15-minutes-between-certain-hours)

In case I am incorrect or something doesn't work, let me know, I can create a separate feature for this :D

**Comment by koppor** [comment]
I don't understand the semantics of `mailInterval`. A) Is the same email sent each 10 seconds to all receipient? B) Is the same email mails once to one receipient and then in next interval to the next one?

Refs https://github.com/adventmail/bulk-mail-cli/issues/27

**Comment by kumarabhirup** [comment]
Yes. It's B). It sends email once and then after the interval to the next one and so on.

By the way, make sure you set the cron string to at least 2 minutes as the cron has hard time working with seconds. Will add this in next update.
